### PR TITLE
Added info about aggregation

### PR DESCRIPTION
**Pull request by letzya**
@marksou the text here is unfortunately duplicated and shows up in another doc. Since I updated it here, might be good to do the same there?

### CODE REVIEW

**Comment by letzya** [comment]
@komalsukhani can you please review? clients and CEs are waiting on this. Thanks

**Comment by komalsukhani** [comment]
@letzya There are some typos. Otherwise it looks good

**Review by komalsukhani — APPROVED** [review]

File: tyk-docs/content/shared/api-def-custom-analytics.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 For example if you include `X-Request-ID` header to tag_headers, for each incoming request it will include a `x-request-id-<header_value>` tag to request an analytic record.
 
 This functionality can be useful if you need to pass additional information from the request to the analytics, without enabling detailed logging, which records the full request and response objects.
+Since we are basically declaring headers' prefix as tags Tyk as expected from tags will also do aggregations per tag for the API calls.
+If you don't want to have aggregation for these tags you can add them or their prefixes to `ignore_tag_prefix_list` in `pump.conf` in case the pump is writing the aggregated analytics to MongoDB. Alternatively, in case MDCB is doing the writing to mongoDB, set the same field in `tyk_sink.conf` at root level (please note that this field is replacing `"aggregates_ignore_tags"` which is still working but will be depricated).
```
Comment (sredxny) [inline tyk-docs/content/shared/api-def-custom-analytics.md]: typo, need to change _depricated_ for _deprecated_

**Comment by letzya** [comment]
thanks @sredxny 
@marksou can we please merge?

**Review by sredxny — APPROVED** [review]

**Pull request merged**